mypy/fastparse2.py

```diff
@@ -469,10 +456,41 @@ def fail_arg(msg: str, arg: None) -> None:
 
         return new_args, decompose_stmts
 
-    def stringify_name(self, n: ast27.AST) -> str:
-        if isinstance(n, ast27.Name):
+    def extract_names(self, arg: ast27.expr) -> List[str]:
+        if isinstance(arg, Name):
+            return [arg.id]
+        elif isinstance(arg, ast27_Tuple):
+            return [name for elt in arg.elts for name in self.extract_names(elt)]
+        else:
+            return []
+
+    def convert_arg(self, index: int, arg: ast27.expr, line: int,
+                    decompose_stmts: List[Statement]) -> Var:
+        if isinstance(arg, Name):
+            v = arg.id
+        elif isinstance(arg, ast27_Tuple):
+            v = '__tuple_arg_{}'.format(index + 1)
```

Comment:
Should we add a test for this? Or do we already have one?

Reply:
We already had tests for Python 2 specific features such as this.